src/main/kotlin/uk/gov/dluhc/emsintegrationapi/rest/PostalVoteApplicationController.kt

```diff
@@ -58,6 +64,22 @@ class PostalVoteApplicationController(private val postalVoteApplicationService:
         logger.info { "DELETE: Processing EMS confirmation of a postal vote application" }
         val serialNumber = authentication.credentials.toString()
         logger.info { "Processing EMS confirmation of a postal vote application with id $applicationId and certificate serial no=$serialNumber" }
-        postalVoteApplicationService.confirmReceipt(serialNumber, applicationId)
+        val request = EMSApplicationResponse(status = EMSApplicationStatus.SUCCESS)
+        postalVoteApplicationService.confirmReceipt(serialNumber, applicationId, request)
+    }
+
+    @PostMapping("/{id}")
```

Comment:
@kristerbone - if it's not too late, can we change this to a PUT?
doing a post on a `/id`  seems strange

Reply:
Now we delete, post and put.